jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/support/AbstractPortsExtractorTest.java

```diff
@@ -100,10 +105,9 @@ public void testNoFile() {
 
     @Test
     public void testConfigFileDoesNotExist() {
+        logger = mock(PrefixedLogger.class);
```

Comment:
Remove above `@Mock`annotation and initialize mock objects like you've done here.